test/hotspot/jtreg/resourcehogs/serviceability/sa/LingeredAppWithLargeArray.java

```diff
@@ -24,8 +24,9 @@
 import jdk.test.lib.apps.LingeredApp;
 
 public class LingeredAppWithLargeArray extends LingeredApp {
+    public static int[] hugeArray;
     public static void main(String args[]) {
-        int[] hugeArray = new int[Integer.MAX_VALUE/2];
+        hugeArray = new int[Integer.MAX_VALUE/2];
         LingeredApp.main(args);
```

Comment:
Another option here is to use Reference.reachabilityFence(hugeArray). That makes it a bit more explicit/obvious that the test needs to keep hugeArray alive.

Reply:
Done! Also changed #7257 to do the same.